### PR TITLE
Webm support

### PR DESCRIPTION
**Pull request by andrenatal**

### CODE REVIEW

File: src/stm.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ var RECORDING_TIMEOUT = 3000;
 var RECORDING_BITS_PER_SECOND = 16000;
 
 var RECORDING_MIME_TYPE = 'audio/ogg';
+if (!MediaRecorder.isTypeSupported(RECORDING_MIME_TYPE)){
+  RECORDING_MIME_TYPE = 'audio/webm;codecs=opus';
+}
```
Comment (autonome) [inline src/stm.js]: Should we just default to webm, as it's more widely supported? What are the reasons for picking one over the other?

File: src/stm.js [hunk 2]
```diff
@@ -180,6 +183,7 @@ function SpeakToMe(options) {
       // create a javascript node
       scriptprocessor = audioContext.createScriptProcessor(
           bufferSize, 1, 1);
+      scriptprocessor.connect(audioContext.destination);
```
Comment (autonome) [inline src/stm.js]: What effect does this change have?

File: src/stm.js (hunk 1 above)
Comment (andrenatal) [reply src/stm.js]: Firefox doesn't support webm, only ogg, whilst Chrome supports webm but not ogg. The goal for this patch is to start supporting both FF and Chrome with the same codebase.

File: src/stm.js (hunk 2 above)
Comment (andrenatal) [reply src/stm.js]: For some reason this was necessary to properly connect `audiocontext` and `scriptprocessor` to make `scriptprocessor.onaudioprocess`  be fired on Chrome. Without the change the event was triggered only on FF.

**Comment by autonome** [comment]
Got it, thanks for explaining!

**Pull request merged**